comicboxd/app/model/book.go

```diff
@@ -35,8 +35,8 @@ type Book struct {
 type UserBook struct {
 	UserID       *uuid.UUID `json:"-"              db:"user_id"`
 	BookID       *uuid.UUID `json:"-"              db:"book_id"`
-	CurrentPage  *int64     `json:"current_page"   db:"current_page"`
-	LastPageRead *int64     `json:"last_page_read" db:"last_page_read"`
+	CurrentPage  *int32     `json:"current_page"   db:"current_page"`
+	LastPageRead *int32     `json:"last_page_read" db:"last_page_read"`
```

Comment:
Why the switch from 64 to 32 bit ints?

Is it because we don't expect the numbers to ever be that high, so 64 is overkill?

Reply:
Graphql uses int32 and the new library requires it